src/icons/EyeIcon.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+import Svg, { Path } from 'react-native-svg'
+import Colors from 'src/styles/colors'
+
+export interface Props {
+  height?: number
+  color?: Colors
+}
+
+function EyeIcon({ color, height }: Props) {
+  return (
+    <Svg height={height} width={height} fill="none" testID="EyeIcon">
```

Comment:
I think this would need a viewBox param set to `0 0 24 24`. Otherwise I don't think the `height` prop makes any difference. Can you test this out by passing a different height than 24?